lib/swaggerUtil.js

```diff
@@ -103,16 +97,16 @@ function validateObject(specs, obj, disc) {
     var validationErrors = [];
     if (!obj[disc]) {//discriminator not defined for this object
         validationErrors.push(
-                {
-                    message: 'discriminator ' + disc + ' not defined for inheriting model ' + JSON.stringify(obj)
-                }
+            {
+                message: 'discriminator ' + disc + ' not defined for inheriting model ' + JSON.stringify(obj)
```

Comment:
I just went ahead and removed it because code in the swagger handler would have had to change in order to accommodate a new property